Update installation.md (#2030)
nuwave/lighthouse: docs/5/getting-started/installation.md

## Lumen

To use the included lighthouse config, copy it over to your config folder.

```bash
mkdir -p config

> cp vendor/nuwave/lighthouse/src/lighthouse config/

Reviewer: ```suggestion
cp vendor/nuwave/lighthouse/src/lighthouse.php config/
```

```

Register the config file within your `bootstrap/app.php` file:

```php
$app->configure('lighthouse');
```

To get you going right away in Lumen, copy over the included default schema.

```bash
mkdir -p graphql
cp vendor/nuwave/lighthouse/src/default-schema.graphql graphql/schema.graphql
```

> Register these service providers in your `bootstrap/app.php` file:

Reviewer: ```suggestion
Register the service provider in your `bootstrap/app.php` file:
```

```php
$app->register(\Nuwave\Lighthouse\LighthouseServiceProvider::class);

> $app->register(\Nuwave\Lighthouse\Pagination\PaginationServiceProvider::class);

Reviewer: ```suggestion
```

The section below describes how to add supplementary service providers

Author: @spawnia Yep, I agree on the sections below specifying supplementary service provides. With the addition of recommending a copying of default-schema.graphql and lighthouse.php (setup parity between Lumen and Full Framework), following the documentation with a fresh install will result in an error on the @paginate directive.

```
No directive found for `paginate`
```

I wouldn't be surprised if others overlook the section below as well :(

Reviewer: This issue prompted me to reconsider the default schema. At first, I wanted to go with a slimmed down version that does not use extra service providers, but found it more useful to have a fully-featured schema that utilizes appropriate features. Feel free to review https://github.com/nuwave/lighthouse/pull/2032.

I would propose to keep this section with the single service provider. Instead, let's move the part about publishing the default schema after it, and then mention the extra required service providers.

Reviewer: Improved with https://github.com/nuwave/lighthouse/commit/6f261ed71112f1ae64fb3088cfeae3a9d4518ad0
